### PR TITLE
Lexi-1045 Adding figma links

### PR DESCRIPTION
**Pull request by victorvalle**
https://issues.liferay.com/browse/LEXI-1045

### CODE REVIEW

**Comment by plhnk** [comment]
running the deploy preview, excited to see this one in action!!

**Review by plhnk — COMMENTED** [review]
nice implementation :) did you read [Kent's post](https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx)?

File: src/markdown/lexicon/core-components/forms/text-input-localizable.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 title: 'Text Input Localizable'
 titleLabel: 'Clay'
 titleLabelLink: 'https://clayui.com/docs/components/localized-input'
+figmaLink: 'https://www.figma.com/file/02DVhuHVTDbzaMjNM7IUKWp0/lexicon?node-id=6033%3A2985
 description: 'A text input variation used in fields that can be translated into multiple languages.'
```
Comment (plhnk) [inline src/markdown/lexicon/core-components/forms/text-input-localizable.md:6]: something is breaking the build here

Comment (plhnk) [reply src/markdown/lexicon/core-components/forms/text-input-localizable.md:6]: ah looks like you have to have a closing quote in figmaLink --> '

**Review by plhnk — CHANGES_REQUESTED** [review]

**Comment by victorvalle** [comment]
I think that at the very beginning when I was trying to modify this code I did tertiary because I couldn't understand well the code. But then you changed it and it pretty and clean for me :). The code was longer and maybe not as beautiful as Ken's code.

We can change if you want. Although I don't know if we have false Trues plotting in our interface :)

**Review by plhnk — COMMENTED** [review]
excited for this!

**Review by plhnk — APPROVED** [review]

**Pull request merged**